docs/en/userguide/index.md

```diff
@@ -0,0 +1,138 @@
+
+Each field can be configured as such:
+
+* The **Original label** is not editable, and serves to indicate the source of the data as coming from the CKAN
+  resource.
+* The **Readable label** allows the label/name to be altered to be more readable if the one imported from CKAN is
+  undesirable for any reason. By default this is automatically generated from the **Original label**.
+* If **Show in results view** is checked, this field will be shown as a column on the field site visitors will see,
+  otherwise it will not be shown. At least one field must be selected to be shown to the user.
+* If **Show in detail view** is checked, this field will be shown in the detail view that opens when a visitor clicks
+  a row on the table shown to them. This is enabled for all fields by default.
+* If **Only show one value if duplicates exist** is checked, removes one of two (or more) results that have the same
+  data in the result set shown to the website visitor.
+* **Presented order** allows an author to change the ordering of columns with more precision than dragging and
+  dropping items in the CMS table. This also makes it easy to move items between pages.
+* The **Result conditions** allow for a "pre-filter" to be chosen to only show results matching (or not matching) a
+  particular value.
+
+## Filters
+
+The **_Filters_** tab allows a CMS content author to configure fields shown to a visitor that will allow them to search
+the results shown to them. One is always created by default after selecting a resource and saving the page. This default
+filter is a text type that searches all columns for matches on the visitor's search term.
+
+By default there are two types of filters available for a content author to configure:
+
+* **Text**: allows a user to enter text to filter results.
+* **Dropdown**: only allows a pre-defined value to be chosen.
+
+### Configuring a text filter
+
+![Configuration options for a text filter](_images/text-filter-config.png)
+
+* **Filter label** provides an appropriate label, e.g. "Search locations".
+* **Column sources** allow the filter to be configured to either search all columns, or if the checkbox is unselected
+  to search only a particular column or set of columns (multiple may be chosen).
+
+### Configuring the dropdown filter
+
+This filter type has only one extra configuration option from those provided by the text filter; the ability to supply
+filter options for the user to choose from. There are two methods available to define options, however only the chosen
+method is used (the two cannot be combined).
+
+#### Options
+
+After choosing which columns the filter will apply to, the **Select from all options** area will update to provide a
+list of all possible unique values those columns contain. The content author can then choose to disable any of these
+options by unselecting the corresponding checkboxes.
+
+![Dropdown filter configuration](_images/dropdown-filter-config.png)
+
+#### Delimiter
```

Comment:
```suggestion
##### Filtering multiple values (Delimiter)
```
Changed to H5, as the content lives under **Select from all options** which is an H4